src/pages/patterns/filters.mdx

```diff
@@ -0,0 +1,416 @@
+---
+
+![Filter pattern basic multiple select 2](figma:filter-usage-basic-multipleselect-2.png)
+
+#### Nested menu
+
+A [Nested menu](/components/menu) can be used to organize subcategories of data that can be used to filter a dataset.
+
+![Filter pattern basic nested menu 1](figma:filter-usage-basic-nestedmenu-1.png)
+
+---
+
+![Filter pattern basic nested menu 2](figma:filter-usage-basic-nestedmenu-2.png)
+
+---
+
+![Filter pattern basic nested menu 3](figma:filter-usage-basic-nestedmenu-3.png)
+
+### Advanced filtering
+
+Advanced Filters allow users to set and configure multiple filters on a dataset.
+
+#### Tooltip modal
+
+A Tooltip modal allows users to filter a dataset through one category with many options,
+and allows the user to search for specific options.
+They can also be part of a more complex flow.
+
+![Filter pattern advanced tooltip modal](figma:filter-usage-advanced-tooltip-modal.png)
+
+#### Sheet
+
+A Sheet is used when the dataset is filtered live as the user makes their selections.
+Selections are surfaced as a Tag as the user sets each filter.
+
+![Filter pattern advanced sheet 1](figma:filter-usage-advanced-sheet-1.png)
+
+---
+
+![Filter pattern advanced sheet 2](figma:filter-usage-advanced-sheet-2.png)
+
+---
+
+![Filter pattern advanced sheet 3](figma:filter-usage-advanced-sheet-3.png)
+
+---
+
+![Filter pattern advanced sheet 4](figma:filter-usage-advanced-sheet-4.png)
+
+---
+
+![Filter pattern advanced sheet 5](figma:filter-usage-advanced-sheet-5.png)
+
+---
+
+![Filter pattern advanced sheet 6](figma:filter-usage-advanced-sheet-6.png)
+
+#### Drawer
+
+A Drawer is used to batch apply multiple filters at once. Selections are surfaced as a Tag
+after the user applies their filters and closes the Drawer.
+
+![Filter pattern advanced drawer 1](figma:filter-usage-advanced-drawer-1.png)
+
+---
+
+![Filter pattern advanced drawer 2](figma:filter-usage-advanced-drawer-2.png)
+
+---
+
+![Filter pattern advanced drawer 3](figma:filter-usage-advanced-drawer-3.png)
+
+## Behavior
+
+### Bulk actions
+
+When items are selected in a table and a filter is applied, the Filter overrides the selections and deselects them.
+
+![Filter pattern behavior bulk actions 1](figma:filter-behavior-bulk-actions-1.png)
+
+---
+
+![Filter pattern behavior bulk actions 2](figma:filter-behavior-bulk-actions-2.png)
+
+## Localization & internationalization
+
+### RTL right-to-left
+
+![Filter pattern localization](figma:filter-localization.png)
+
+Filter entry points are mirrored for RTL languages.
+For more information about RTL considerations, see our [Buttons pattern](/patterns/buttons).
+
+## Accessibility
+
+### Tabbing order
+
+![Filter pattern accessibility tabbing order](figma:filter-accessibility-tabbing.png)
+
+1. User opens the filter container
+2. The focus moves to the first filter category and continues through all categories.
+3. If filters are applied live, focus moves to close filter. Filter can also be closed with the `esc` key.
```

Comment:
This is currently labeled 2️⃣ in the Figma graphic; I think it should be 3️⃣ .

Reply:
Swap 2 and 3?

Reply:
See updated Accessibility section

Reply:
This still isn't looking right to me. As with all `Sheet` components, by design the close `X` is the _last_ item to get keyboard focus within the modal. See https://garden.zendesk.com/components/sheet and observe the keyboard tab order for the examples.